src/main/features/response/routes/your-payment-plan.ts

```diff
@@ -0,0 +1,48 @@
+import * as express from 'express'
+
+import { Paths } from 'response/paths'
+
+import { ErrorHandling } from 'common/errorHandling'
+import { Form } from 'app/forms/form'
+import { DraftService } from 'services/draftService'
+import { User } from 'idam/user'
+import { PaidAmount } from 'ccj/form/models/paidAmount'
+import { DefendantPaymentPlan } from 'response/form/models/defendantPaymentPlan'
+import { FormValidator } from 'forms/validation/formValidator'
+
+function renderView (form: Form<PaidAmount>, res: express.Response): void {
+  const user: User = res.locals.user
+  const alreadyPaid: number = user.responseDraft.document.paidAmount.amount || 0
```

Comment:
Why do you need to fallback to 0 here? Is it possible paid amount will not be available at this point?

Reply:
Well having removed the || 0, the amount doesn't show at all

Reply:
Ok, it's fine 👍